{
  "comments": [
    {
      "key": {
        "uuid": "588c60bc_7cb90224",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 559,
      "author": {
        "id": 1161379
      },
      "writtenOn": "2020-04-09T02:03:37Z",
      "side": 1,
      "message": "use for loop here?",
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "81fcd70d_fa4c730d",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 570,
      "author": {
        "id": 1366072
      },
      "writtenOn": "2020-04-09T02:15:54Z",
      "side": 1,
      "message": "Is this max timeout reasonable?\nI\u0027m not sure how long it should be. but 50s minute would be long?",
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "5bfdb8d4_3b6827c5",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 570,
      "author": {
        "id": 1161379
      },
      "writtenOn": "2020-04-09T02:19:12Z",
      "side": 1,
      "message": "This should basically not happen, so I think it is better to have sufficiently enough buffer here as long as task timeout doesn\u0027t happen?",
      "parentUuid": "81fcd70d_fa4c730d",
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "b2dbf476_f025396c",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 573,
      "author": {
        "id": 1161379
      },
      "writtenOn": "2020-04-09T02:03:37Z",
      "side": 1,
      "message": "use proc.wait() after terminate and kill too like\nhttps://cs.chromium.org/chromium/infra/luci/client/utils/subprocess42.py?l\u003d22\u0026rcl\u003dd2b3bc2b653aef732961539e51c1b9d183bc45e7",
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "fdcbadd1_c08a2b00",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 574,
      "author": {
        "id": 1375000
      },
      "writtenOn": "2020-04-09T02:02:29Z",
      "side": 1,
      "message": "Maybe i shouldn\u0027t break here? This way it goes another round of loop --\u003e retcode !\u003d 0 --\u003e raise ValueError, which is the expected way of how it work? Or re-raise?",
      "range": {
        "startLine": 574,
        "startChar": 8,
        "endLine": 574,
        "endChar": 13
      },
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "fe3a8bae_668b92ef",
        "filename": "client/run_isolated.py",
        "patchSetId": 3
      },
      "lineNbr": 574,
      "author": {
        "id": 1161379
      },
      "writtenOn": "2020-04-09T02:04:36Z",
      "side": 1,
      "message": "yeah, I think we need to raise here.",
      "parentUuid": "fdcbadd1_c08a2b00",
      "range": {
        "startLine": 574,
        "startChar": 8,
        "endLine": 574,
        "endChar": 13
      },
      "revId": "96f54601a6ca17d24c5980241ed8dac7a6a2189d",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    }
  ]
}